modules/dataframe/include/inviwo/dataframe/datastructures/dataframe.h

```diff
@@ -211,6 +220,14 @@ std::shared_ptr<TemplateColumn<T>> DataFrame::addColumn(const std::string &heade
     return col;
 }
 
+template <typename T>
+std::shared_ptr<TemplateColumn<T>> DataFrame::addColumn(const std::string &header,
+                                                        const std::vector<T> &data) {
+    auto col = std::make_shared<TemplateColumn<T>>(header);
```

Comment:
To avoid the copy it this would be better:
```template <typename T>
std::shared_ptr<TemplateColumn<T>> DataFrame::addColumn(const std::string &header,
                                                        std::vector<T> data) {
    auto ram = std::make_shared<BufferRAMPrecision<T>>(std::move(data));
    auto buf = std::make_shared<Buffer>(ram);
    auto col = std::make_shared<TemplateColumn<T>>(header, buff);
    columns_.push_back(col);
    return col;
}
```